src/python_framework_v02/nhd_network_utilities_v02.py

```diff
@@ -267,7 +267,7 @@ def set_supernetwork_data(
     elif supernetwork == "CONUS_FULL_RES_v20":
 
         ROUTELINK = "RouteLink_NHDPLUS"
-        ModelVer = "nwm.v2.0.2"
+        ModelVer = "nwm.v2.0.3"
```

Comment:
Needed to change this because version has been updated online. nw,.v2.0.2 no longer exists. Without this change, any existing code that automatically downloads the NHD dataset will not work.

Reply:
Addressed in #133, Fixed in #134